dist/index.js

```diff
@@ -1062,11 +1088,19 @@ function loadConfig() {
         if (!fs_1.default.existsSync(filename)) {
             core.setFailed(`Could not find pipeline at ${baseFolder}/${pipeline}`);
         }
+        core.info(`file parameters ${runtimeParametersFile}`);
+        core.info(`folder parameters ${folderName}`);
+        if (runtimeParametersFile) {
+            core.info("Reading runtime parameters file");
+            runtimeParametersFile = readParemetersFile(folderName, pipeline, runtimeParametersFile);
```

Comment:
This method rather than a file name, return the content of the file. So I think you should use a different variable name here and in the config.

Reply:
I just update the method, I think now is more clear